pages/common/crystal.md

```diff
@@ -0,0 +1,23 @@
+# crystal
+
+> Crystal language tool for compiling.
+
+- Run a crystal file:
+
+`crystal {{path/to/file.cr}}`
+
+- Compile a file and all dependencies to a single executable:
+
+`crystal build {{path/to/file.cr}}`
+
+- Start a local interactive server for testing the language:
+
+`crystal play`
+
+- Create a project directory for a crystal application:
+
+`crystal init app {{name of application}}`
```

Comment:
Can we have a concrete example here instead of a generic one?

Reply:
Just to note, this should be changed to token syntax like `name_of_application` (or `application_name`). Unless it gets changed to a concrete example. :thinking:

Reply:
Would `crystal init app my_blog` serve as a more concrete example?